lib/libutee/include/pta_system.h

```diff
@@ -13,6 +13,17 @@
 #define PTA_SYSTEM_UUID { 0x3a2f8978, 0x5dc0, 0x11e8, { \
 			 0x9c, 0x2d, 0xfa, 0x7a, 0xe0, 0x1b, 0xbe, 0xbc } }
 
+/*
+ * Having keys with too few bits impose a potential security risk, hence set a
+ * lower bound (i.e., 128 bits).
+ */
+#define TA_DERIVED_KEY_MIN_SIZE 16
```

Comment:
tab before macro value, here and below.

Reply:
Sure, for consistency with existing defines I'll update, but otherwise do we have a rule for that?

Reply:
no, I don't think so.
I believe indentation here helps a bit reading back the macro value.

Reply:
Will fix.